tests/unit/composables/filters/Filter.test.js

```diff
@@ -0,0 +1,48 @@
+import Filter from 'src/composables/filters/Filter';
+
+describe('Test class: Filter', () => {
+  let filter;
+
+  beforeEach(() => {
+    filter = new Filter('testFilter', 'serverTestFilter', 'queryTestFilter', 'default');
+  });
+
+  describe('Test constructor', () => {
+    it('should create an instance of Filter', () => {
+      expect(filter).toBeInstanceOf(Filter);
+      expect(filter.filterName).toBe('testFilter');
+      expect(filter.serverName).toBe('serverTestFilter');
+      expect(filter.queryName).toBe('queryTestFilter');
+      expect(filter.defaultValue).toBe('default');
+    });
+  });
+
+  describe('Test function: getFilterValue', () => {
+    it('should return the value passed to it', () => {
+      const value = 'someValue';
+
+      expect(filter.getFilterValue(value)).toBe(value);
+    });
+  });
+
+  describe('Test function: getValueFromUrl', () => {
+    it('should return the value passed to it', () => {
+      const value = 'urlValue';
+      expect(filter.getValueFromUrl(value)).toBe(value);
+    });
+  });
+
+  describe('Test function: isDefaultValue', () => {
+    it('should return true for default value', () => {
```

Comment:
bad test, because we can only have one default value, '' or null or undefined is not the same value.

Add test for these values: `0`, `false`, `''` and `'test'`

Reply:
done